pages/docs/support/rest-api.mdx

```diff
@@ -0,0 +1,110 @@
+---
+title: "Grouparoo REST API"
+date: "2021-08-02"
+pullQuote: "Use the Grouparoo REST API to connect to your Grouparoo instance from an external application."
+---
+
+Grouparoo's REST API enables you to connect to your Grouparoo instance from an external application and return data in JSON format.
+
+## What can you do with our API?
+
+Grouparoo's REST API contains endpoints for over 25 topics -- everything from installing a Grouparoo Plugin to getting a list of all your Schedules. To see a full list of API endpoints, navigate to `/swagger` in the UI of your Grouparoo instance or run:
+
+```bash
+curl -X 'GET' \
+  '{{yourinstanceurl}}/api/v1/swagger' \
+  -H 'accept: application/json'
+```
```

Comment:
I think it's a little odd to use the Swagger endpoint to learn about the API.  I like telling folks to visit `/swagger` in the UI.  We can also suggest they use a tool like Postman (it can import swagger)?

<img width="1651" alt="Screen Shot 2021-08-03 at 5 38 20 PM" src="https://user-images.githubusercontent.com/303226/128103603-2d608a65-cc7a-46b3-9306-1e38a0b50854.png">